Sources/SwiftSMTP/Mail.swift

```diff
@@ -122,21 +122,21 @@ public struct Mail {
         dictionary["MESSAGE-ID"] = id
         dictionary["DATE"] = Date().smtpFormatted
         dictionary["FROM"] = from.mime
-        dictionary["TO"] = to.map { $0.mime }.joined(separator: ", ")
+        dictionary["TO"] = (to.map { $0.mime }.joined(separator: ", ")).wrap()
```

Comment:
If there was a `wrap()` method on Array, we can replace `joined(separator: ", ").wrap()` with a single `wrap()` call that will be able to ensure comma-separated lists are split at the comma boundaries.  While not required, it will make the header more human readable.

From the RFC:
> if a field body is defined as comma-separated values, it is recommended that folding occur after the comma